cypress/support/commands.js

```diff
@@ -28,6 +28,10 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 
 addMatchImageSnapshotCommand();
 
+Cypress.Commands.add('getByDataQa', (selector) => {
+  cy.get(`[dataqa="${selector}"]`);
```

Comment:
cy.get query is not returning anything. In Cypress, the .get command should always return an element or elements. The current usage will not work as expected.